lib/tests/tests.c

```diff
@@ -956,6 +962,60 @@ test_demographic_events(void)
     tsk_table_collection_free(&tables);
 }
 
+static void
+test_census_event(void)
+{
+    int ret;
+    uint32_t n = 10;
+    sample_t *samples = malloc(n * sizeof(sample_t));
+    msp_t *msp = malloc(sizeof(msp_t));
+    gsl_rng *rng = gsl_rng_alloc(gsl_rng_default);
+    recomb_map_t recomb_map;
+    tsk_table_collection_t tables;
+    int no_census_nodes = 0;
```

Comment:
I read this as "no census nodes``, ie, as a boolean which would be true if there were no census nodes. ``num_census_nodes`` would follow the local conventions better.